media/css/pebbles/components/_modal.scss

```diff
@@ -18,6 +18,18 @@ html.noscroll {
     #modal {
         position: absolute;
     }
+
+    @media #{$mq-tablet} {
+        height: auto;
+
+        body {
+            height: auto;
+        }
+
+        #modal {
+            position: fixed;
```

Comment:
I noticed that setting `body` to `position: relative;` also fixes the display issue, but I think this fix is probably better to make the modal behavior consistent 👍